conf/application.conf

```diff
@@ -181,6 +181,7 @@ tracking-consent-frontend {
 
 features {
   welsh-translation: true
+  traderDetailsCacheEnabled: false
```

Comment:
This flag was copied verbatim from another pull request, so the name is totally misleading. I'll change it to something that better describes the feature.

Reply:
Done